routes/movies.routes.js

```diff
@@ -0,0 +1,105 @@
+const Movie = require("../models/Movie.model");
+const Celebrity = require("../models/Celebrity.model");
+const router = require("express").Router();
+
+//READ movies
+router.get("/movies", (req, res, next) => {
+    Movie.find()
+        .then((moviesArr) => {
+            res.render("movies/movies", {
+                movies: moviesArr
+            })
+        })
+        .catch((err) => {
+            console.log("Oops, there was an error!", err)
+        })
+
+})
+
+//CREATING new movies
+
+router.get("/movies/create", (req, res, next) => {
+    res.render("movies/new-movie")
+})
```

Comment:
in this GET route, you have to get all celebrities from database in order to have a dropdown menu when creating a movie

So you have to access celebrities model here `Celebrity.find()`